clang/test/SemaSYCL/intel-fpga-local.cpp

```diff
@@ -442,20 +442,20 @@ void check_gnu_style() {
   int __attribute__((force_pow2_depth(0))) force_p2d;
 }
 
-//expected-error@+1{{attribute only applies to local non-const variables and non-static data members}}
+//expected-error@+1{{'private_copies' attribute only applies to const variables, local variables, non-static data members and device_global variables}}
 [[intel::private_copies(8)]]
 __attribute__((opencl_constant)) unsigned int const_var[64] = {1, 2, 3};
 
 void attr_on_const_error()
 {
-  //expected-error@+1{{attribute only applies to local non-const variables and non-static data members}}
+  //expected-no-error@+1
   [[intel::private_copies(8)]] const int const_var[64] = {0, 1};
 }
```

Comment:
This is OK now. I have removed this restriction as per discussion with @artemrad